synapse/models/inet.py

```diff
@@ -102,6 +103,41 @@ def norm(self, valu, oldval=None):
     def repr(self, valu):
         return valu.encode('utf8').decode('idna')
 
+class Rfc2822Addr(DataType):
+    '''
+    An RFC 2822 compatible email address parser
+
+    Specify strict=1 to strictly enforce complaince.
+    '''
+    def norm(self, valu, oldval=None):
+
+        # remove quotes for normalized version
```

Comment:
Probably want to make sure we are only norming a str and raise bad type value otherwise

Reply:
Good call, i'll add it for clarity.